app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java

```diff
@@ -153,6 +156,8 @@ public void onAttach(Context context) {
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
 
+        mSharedPreferences = PreferenceManager.getDefaultSharedPreferences(mActivity);
+        mPageNumStyle = mSharedPreferences.getString (Constants.PREF_PAGE_STYLE, null);
```

Comment:
move this line to `resetValues` function

Reply:
this isnt done yet

Reply:
So I should remove the number 160 line to resetValues?